types/chai-datetime/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/chai-datetime",
-    "version": "0.0.9999",
+    "version": "1.0.9999",
```

Comment:
This library is actually at 1.8; not sure if it's valuable to fix that now or not. 1.0 is okay.

Reply:
I think I did not go all the way to 1.8 because I did not have the time to see if anything unrelated might have changed